app/js/components/visitsAndEncounters.jsx

```diff
@@ -307,6 +315,14 @@ export class VisitsAndEncounters extends React.Component {
       id: 'visit',
       Header: 'Visit',
       accessor: rowProps => rowProps.visit,
+      getProps: (state, rowInfo, column) => {
+        return {
+          style: {
+            textDecoration: rowInfo.row.voided === 'true' ? 'line-through' : null 
```

Comment:
same goes for this

Reply:
@brayoh thanks for the pointer. However, 'true' here is a string value and not boolean.

Reply:
javascript has type coercion which means so for any primitive or object type value that's not undefined or null it evaluates to true.

Reply:
It comparing a string against a string not a boolean @brayoh

Reply:
sorry, I missed out on that, I thought it was boolean.